arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json

```diff
@@ -0,0 +1,305 @@
+{  
+   "swagger":"2.0",
+   "info":{  
+      "title":"Azure SQL Database API spec",
+      "description":"The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your external server administrators.",
+      "version":"2014-04-01"
+   },
+   "host":"management.azure.com",
+   "schemes":[  
+      "https"
+   ],
+   "consumes":[  
+      "application/json"
+   ],
+   "produces":[  
+      "application/json"
+   ],
+   "paths":{  
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/administrators/activedirectory":{  
```

Comment:
This path looks wrong to me, having the fixed "activedirectory" at the end.  Shouldn't it be something like "{administratorName}"?  @ravbhatnagar is this ok per ARM rules?